src/description.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+var fs = require('fs');
+var path = require('path');
+var s = require('util').format;
+var marked = require('marked');
```

Comment:
In my example, I want to highlight the fact that I use a `isFile` function instead of the current `isPath` doing regex-based matching (for `.md` and `.markdown` files).

This would allow to provide a description path that is not Markdown, and still have it read the file because the file actually exists. In combination with optional Markdown filter (not tied to `description`, but contained in `markdown`), one could make a custom AsciiDoc filter and run `extras('description', asciidoc)` and have it working as expected.